src/test/java/com/wikia/webdriver/testcases/desktop/articlecrudtests/ArticleSourceModeTests.java

```diff
@@ -280,4 +287,72 @@ public void RTE_019_MoreSympolsTools() {
     source.checkSymbolsTools();
     source.submitArticle();
   }
+
+  @Test(groups = {"RTE_draft","draft_saving"})
+  @Execute(asUser = User.SUS_REGULAR_USER3, onWikia = "draftsavetest")
+  public void RTE_draft_save() {
+    String articleName = PageContent.ARTICLE_NAME_PREFIX + DateTime.now().getMillis();
+    new ArticleContent(User.SUS_REGULAR_USER3).push("An awesome start", articleName);
+    SourceEditModePageObject source = new SourceEditModePageObject().openArticle(articleName);
+    source.addContentInSourceMode(articleName);
+//This sleep is necessery for draft to be saved (every 5 sec)
+    try {
```

Comment:
Test side should be deterministic. It'd be better to move this try/catch and refreshing to a separate method. In BasePageObject, I guess, since it's very general.

Reply:
hmm, might be ok to move to  waitForDraftToBeSaved() method?